src/components/Feedback.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import Tabs from "react-bootstrap/Tabs";
+import Tab from "react-bootstrap/Tab";
+import axios from "axios";
+import "../css/feedbackpage.css";
+import CommentBox from "./comments/CommentBox";
+import { LOCAL_HOST } from "../consts";
+import QRTab from "./feedback-tabs/QRTab";
+import DownloadTab from "./feedback-tabs/DownloadTab";
+
+class Feedback extends Component {
```

Comment:
Please rename the component. It's the project page, not just the feedback. And can't this be inside `CommentsPage.jsx`? The only component inside `CommentsPage` is `Feedback` component.

Reply:
I agree with the first point of renaming.

I don't with the second point, however. You can't put this section in the `CommentsPage` because you need `CommentsPage` to be its own modular component with the Tabs; which isn't possible to do if you don't have this separate module.

Reply:
Done